storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc

```diff
@@ -10,7 +10,6 @@ toc::[]
 {product-title} can provision persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Cloud Platform (GCP) persistent disk (PD) storage.
 
 :FeatureName: GCP PD CSI Driver Operator
-include::modules/technology-preview.adoc[leveloffset=+1]
 
 Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a Container Storage Interface (CSI) Operator and driver.
 
```

Comment:
@jsafrane A little later in this section (line 24), there is a note for the driver:

> OpenShift Container Platform defaults to using an in-tree, or non-CSI, driver to provision GCP PD storage. This in-tree driver will be removed in a subsequent update of OpenShift Container Platform. Volumes provisioned using the existing in-tree driver are planned for migration to the CSI driver at that time.

Is this ^^ note still valid in 4.8?

Reply:
Yes, the note is still valid. Current migration date is OCP 4.10, the note is needed until then.